package.json

```diff
@@ -16,7 +16,7 @@
     "test": "make test"
   },
   "dependencies": {
-    "browser-launcher": "^1.0.1",
+    "browser-launcher": "git+https://github.com/substack/browser-launcher.git#603e1ea",
```

Comment:
not everyone has git+https configured (vs ssh), can you change this url to `subtack/browser-launcher#603e1ea`? Then I think npm will figure out the right protocol for you